cypress/tests/basic-auth.cy.ts

```diff
@@ -1,29 +1,23 @@
-const PAGE_URL = 'basic_auth';
-const AUTH_CREDS = {
-  username: 'admin',
-  password: Cypress.env('BASIC_AUTH_PASSWORD')
-};
-const BAD_CREDS = {
-  username: 'wrong',
-  password: 'creds'
-};
+import { BasicAuthPage } from '../support/page-objects/basic-auth.po';
 
 describe('Basic Auth Page Functionality', () => {
   it('Should be able to access page with correct credentials', () => {
-    // Visit page with authentication options loaded in (cannot interact with login browser prompt)
-    cy.visit(PAGE_URL, { auth: AUTH_CREDS });
+    // Visit page with correct authentication options loaded in (cannot interact with login browser prompt)
+    BasicAuthPage.visitSuccess();
 
     // Assert successful authentication by visibility of the contents on the page
-    cy.get('h3').contains('Basic Auth').should('be.visible');
-    cy.get('#content > .example > p')
+    BasicAuthPage.getHeader()
+      .should('contain.text', 'Basic Auth')
```

Comment:
Header and body text should be moved to page object, to keep in line with A/B Testing spec/po changes